drivers/SmartThings/sonos/src/ssdp.lua

```diff
@@ -8,9 +8,24 @@ SONOS_SSDP_SEARCH_TERM = "urn:smartspeaker-audio:service:SpeakerGroup:1"
 local SSDP = {}
 
 local function process_response(val)
-  local info = {}
+  -- check first line assuming it's the HTTP Status Line, which if not is invalid
+  local status_line = string.match(val, "([^\r\n]*)\r\n")
+  if not (status_line and string.match(status_line, "HTTP/1.1 200 OK"))  then
```

Comment:
Does it need to be `HTTP/1.1` here or could it be a different version and still work?

Reply:
UPnP/SSDP is always HTTP/1.1: https://openconnectivity.org/upnp-specs/UPnP-arch-DeviceArchitecture-v2.0-20200417.pdf

Section 1.1